cmd/skopeo/proxy.go

```diff
@@ -266,6 +268,23 @@ func (h *proxyHandler) openImageImpl(args []any, allowNotFound bool) (replyBuf,
 		return ret, err
 	}
 
+	unparsedTopLevel := image.UnparsedInstance(imgsrc, nil)
+	policy, err := signature.DefaultPolicy(h.sysctx)
```

Comment:
While I think this is OK for now, maybe it makes sense to cache this in the future?

Reply:
Ideally, cache the `PolicyContext` (as long as it is used from a single thread).

(Right now that does not make a difference, but a branch caching the imported keys in a native format inside a `PolicyContext` is floating around.)